README.md

```diff
@@ -22,10 +35,38 @@ Make sure `$GOPATH/bin` is in your `$PATH`.
       -token string
         	github access token
 
-You'll need a Github access token, which you can generate
-[here](https://github.com/settings/tokens). Dave only needs `repo` scope to
-work.
-
 All flags have an environment variable that sets the default, so setting
 `DAVE_REPO` also changes the repo Dave looks at. Flags override environment
 variables.
+
+## Contributing
+
+See the [CONTRIBUTING.md](CONTRIBUTING.md) file on how to contribute to this project.
+
+## Contributors
+
+See the [CONTRIBUTORS.md](CONTRIBUTORS.md) file for a list of contributors to the project.
+
+## Roadmap
```

Comment:
This is the default template we use. If I where to change it here I need to ask 10 other projects to change theirs as well. This is also based on what should be in a readme from the research.

I could remove the roadmap ofcourse if there is none